idaes_examples/notebooks/docs/flowsheets/temperature_swing_adsorption/temperature_swing_adsorption.ipynb

```diff
@@ -0,0 +1,638 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "tags": [
+     "header",
+     "hide-cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "###############################################################################\n",
+    "# The Institute for the Design of Advanced Energy Systems Integrated Platform\n",
+    "# Framework (IDAES IP) was produced under the DOE Institute for the\n",
+    "# Design of Advanced Energy Systems (IDAES).\n",
+    "#\n",
+    "# Copyright (c) 2018-2023 by the software owners: The Regents of the\n",
+    "# University of California, through Lawrence Berkeley National Laboratory,\n",
+    "# National Technology & Engineering Solutions of Sandia, LLC, Carnegie Mellon\n",
+    "# University, West Virginia University Research Corporation, et al.\n",
+    "# All rights reserved.  Please see the files COPYRIGHT.md and LICENSE.md\n",
+    "# for full copyright and license information.\n",
+    "###############################################################################"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# TSA Adsorption Cycle for Carbon Capture\n",
+    "\n",
+    "## Learning outcomes\n",
+    "\n",
+    "\n",
+    "- Demonstrate the use of the IDAES fixed bed temperature swing adsorption (TSA) 0D unit model\n",
+    "- Initialize the IDAES fixed bed TSA 0D unit model\n",
+    "- Simulate the IDAES fixed bed TSA 0D unit model by solving a square problem\n",
+    "- Generate and analyze results\n",
+    "\n",
+    "\n",
+    "## Problem Statement\n",
+    "\n",
+    "This Jupyter notebook shows the simulation of a fixed bed TSA cycle for carbon capture by using the fixed bed TSA 0D unit model in IDAES. The fixed bed TSA model consists of a 0D equilibrium-based shortcut model composed of four steps a) heating, b) cooling, c) pressurization, and d) adsorption. Note that the equations in the IDAES fixed bed TSA 0D unit model and the input specifications used in this tutorial for the feed stream have been taken from <a href=\"https://pubs.acs.org/doi/abs/10.1021/ie5048829\">Joss et al. 2015</a>.\n",
+    "\n",
+    "\n",
+    "#### A diagram of the TSA adsorption cycle is given below: \n",
+    "\n",
+    "![](tsa_cycle.svg)\n"
```

Comment:
Minor detail, if not too difficult to update it would be nice if the text all faced the same direction. The "CO2" stream label faces opposite the rest of the vertical text.

Reply:
changed